aiida/orm/nodes/data/upf.py

```diff
@@ -269,6 +269,11 @@ class UpfData(SinglefileData):
     """`Data` sub class to represent a pseudopotential single file in UPF format."""
 
     def __init__(self, file=None, source=None, **kwargs):
```

Comment:
Shouldn't we also then expose `filename` on the constructor of this sub class? Same goes for `CifData`.

Reply:
oops - forgotten.
I got sidetracked by this `source` variable that I had never encountered before.

Reply:
also add it for `CifData` please